fixtures/stairville/mh-x50.json

```diff
@@ -1,669 +1,666 @@
 {
-  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
-  "name": "MH-X50+",
-  "categories": ["Moving Head", "Color Changer"],
-  "meta": {
-    "authors": ["Hannes Rüger"],
-    "createDate": "2020-07-16",
-    "lastModifyDate": "2020-07-16"
-  },
-  "links": {
-    "manual": [
-      "https://images.static-thomann.de/pics/atg/atgdata/document/manual/250558_c_250558_v3_en_online.pdf"
-    ],
-    "productPage": [
-      "https://www.thomann.de/intl/stairville_mhx50_led_spot_moving_head.htm"
-    ],
-    "video": [
-      "https://video2.thomann.de//vidiot/02591c1c/video_i2514p10_yd59vqpa.mp4"
-    ]
-  },
-  "physical": {
-    "dimensions": [240, 370, 280],
-    "weight": 10.3,
-    "power": 135,
-    "DMXconnector": "3-pin",
-    "bulb": {
-      "type": "50W LED",
-      "lumens": 940
+    "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
```

Comment:
@hrueger Could you please reset the indentation to two spaces?

Reply:
Hm, my VSCode is ignoring my indentation settings, weird...
However, should be fixed now.